pages/entropy/contract-addresses.mdx

```diff
@@ -0,0 +1,21 @@
+# Entropy on EVM
```

Comment:
All contract addresses have been pulled out of the single per-chain document and moved to a separate contract address reference page